### PR TITLE
Create adding-new-page-custom-url.md

### PR DESCRIPTION
**Pull request by diana-lakatos**
Tutorial for the Get Started/Pages section.

### CODE REVIEW

File: _PlatformOS/getting-started/pages/adding-new-page-custom-url.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+{% raw %}
+
+```liquid
+---
+slug: /
+---
+
+<br><br>
+<a href_"/about">About</a>
```
Comment (pavelloz) [inline _PlatformOS/getting-started/pages/adding-new-page-custom-url.md]: Typo: `href=`

File: _PlatformOS/getting-started/pages/adding-new-page-custom-url.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+{% raw %}
+
+```liquid
+---
+slug: about
+---
+
+About page
+
+<a href_"/">Home</a>
```
Comment (pavelloz) [inline _PlatformOS/getting-started/pages/adding-new-page-custom-url.md]: same typo ^

File: _PlatformOS/getting-started/pages/adding-new-page-custom-url.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+Add a link to the previously created `home.liquid` page, that links to the About page: 
+
+{% raw %}
+
+```liquid
+---
+slug: /
+---
+
+<br><br>
```
Comment (pavelloz) [inline _PlatformOS/getting-started/pages/adding-new-page-custom-url.md]: I think for the sake of this example `<br><br>` is unnecessary

**Comment by diana-lakatos** [comment]
@pavelloz Thank you for your review! I fixed the typos and removed the <br><br> tags.

**Comment by pavelloz** [comment]
👍

**Pull request merged**